Libraries/VRLayers/CylindricalPanel.js

```diff
@@ -43,8 +43,8 @@ const requireNativeComponent = require('requireNativeComponent');
  * </CylindricalPanel>
  * ```
  *
- * The width and height of the cylinder must be specified, and the width must also correspond to
- * the arc that the cylinder covers. The default Density is 4680px, which has been empirically
+ * The width and height of the cylinder must be specified, and the width must also corresponds to
```

Comment:
I think "correspond" was correct initially

Reply:
Good catch, fixed it in https://github.com/facebook/react-vr/pull/385/commits/d300f994444e8d91b28840413ffa62cd8d05ebea Thanks!